vcluster/deploy/topologies/air-gapped.mdx

```diff
@@ -324,17 +324,21 @@ Use Helm to install the vCluster Pro instance into the `namespace` where you ins
 
 ```bash title="Install vCluster Pro with Helm"
 export VCLUSTER_PRO_NAME="vcluster-pro"
+export VCLUSTER_PRO_NAMESPACE="vcluster-ns"
 helm upgrade --install "${VCLUSTER_PRO_NAME}" vcluster-${VERSION}.tgz \
   --version ${VERSION} \
   --values vcluster.yaml \
-  --namespace vcluster-ns
+  --namespace ${VCLUSTER_PRO_NAMESPACE} \
+  --create-namespace
 ```
 
 :::tip Optionally increase resource limits
-If necessary, increase resources limits on the vCluster Pro syncer (control plane `StatefulSet`) pod:
+If necessary, increase resources limits on the vCluster Pro pod for the syncer
```

Comment:
I am not sure I understand why this - increase resource is needed for an airgapped setup. Why will there be a increase in limits needed for an airgapped setup?

Reply:
This part is now removed.